readme.md

```diff
@@ -1,8 +1,8 @@
 GS1 barcode generator
 =====================
 
-The library generates GS1 barcode. Currently supported formats: GS1 128. Code is under development so it may change 
-radically. If you found an issue just let me know. 
+The library generates GS1 barcode. Currently supported formats: GS1 128. Code is under development so it may change
+radically. If you found an issue just let me know.

```

Comment:
I'm using Atom in Linux and it changed some line endings in this file. Sorry for the mess.